python/paddle/fluid/tests/unittests/test_fc_mkldnn_op.py

```diff
@@ -60,37 +67,16 @@ def test_check_output(self):
         self.check_output()
 
     def test_check_grad_normal(self):
-        self.check_grad(set(['Input', 'W']), 'Out', max_relative_error=0.9)
+        pass
 
     def test_check_grad_no_weight(self):
-        self.check_grad(
-            ['Input'], 'Out', max_relative_error=0.5, no_grad_set=set('W'))
+        pass
```

Comment:
It seems that these tests are omitted. Why?

Reply:
This pass was designated for inference, while these disabled tests validate the backward pass.